utils/build.ps1

```diff
@@ -1668,8 +1668,15 @@ function Build-Foundation([Platform]$Platform, $Arch, [switch]$Test = $false) {
     }
 
     $env:CTEST_OUTPUT_ON_FAILURE = 1
+    if ($env:CI) {
+      # Use the windows-specific checkout on CI that provides
+      # a checkout that does not yet use swift-foundation.
+      $RepoName = "swift-corelibs-foundation-windows";
+    } else {
+      $RepoName = "swift-corelibs-foundation";
```

Comment:
Where does the script that specifies `swift-corelibs-foundation` for local builds live? Is it currently checking out `main` or `windows/main`?

Reply:
That is at https://github.com/compnerd/swift-build/blob/main/default.xml#L20. It should checkout `windows/main` for now and will need to be changed back to `main` once main restores the ability to build for Windows.